client/battle/CCreatureAnimation.cpp

```diff
@@ -142,6 +142,19 @@ void CCreatureAnimation::setType(CCreatureAnim::EAnimType type)
 	play();
 }
 
+void CCreatureAnimation::shiftColor(ColorShifter * shifter)
+{
+	if (forward)

```

Comment:
1. No space after `if` please (also after `for` in other files)
2. Do not use `{}` around single statement

Reply:
Updated.